### DISCUSSION TITLE
Amiga 2000 EATX Computer Case Discussion

### DISCUSSION THREAD

**Discussion by jasonsbeer** [jasonsbeer/Amiga-2000-ATX#4] [Ideas]
I thought it would be useful to start a discussion around what computer cases people are using for their Amiga 2000 EATX. There are so many options out there, it may be helpful to share your experiences with different cases.

If you post, please be sure to discuss your system configuration so people can compare against what they want for their system. Be sure to list some pros and cons of your case. Pictures would be great.

I'll start...

For the first build I used a **Phanteks Enthoo** case. 

**Pros:** 

- It handles full length zorro and CPU cards no problem. 
- It has two external 5.25" drive bays that work for either a gotek or floppy drive (with adapter)

**Cons:** 

- My only dislike about this case is that it is HUGE.

**Comment by neilo40** [comment]
Just wanted to say I’ve been following this project since it’s inception and can’t wait to build one. Thank you so much for creating it

I have my eye on the GD08 from silverstone https://www.silverstonetek.com/product.php?pid=331&area=en

I haven’t double checked the dimensions but I like the look. It even reminds me of the original a2000 a little

![](https://www.silverstonetek.com/images/products/gd08/GD08-34View-02.jpg)

### pros:

* two 5.25” bays
* Classic a2000 desktop form factor
* Should be large enough for full length zorro boards (345mm)

### cons:
* nothing obvious yet

**Comment by darrenfreeman** [comment]
I'm unsure about what case to get. I figured I'll get the board working on the bench, and then I'll have a better idea of what fits and what doesn't. (I don't have a feel for how EATX differs from ATX.)

I had a quick look at a $50 "gaming" tower case online, and I think it would be fine.

In fact, this is the first build that I would want to show off with a transparent side cover. (Although, I might end up putting some mesh under that panel, as EMC is still a thing.)

**Comment by thepillock** [comment]
I wanted a retro look and feel for mine, plus drive bays which a lot of modern EATX cases seem to not feature.
After much squinting at eBay pictures counting mounting holes (EATX have four columns, ATX is three is quite an easy way to tell) I bought a Fujitsu Scenic XL case with a tinted green slide-down front and this mainboard fits really well. The only issue might be that Gary is in the way of a full-depth CD drive in the lowest drive bay, but there are two above that are unhindered plus you'd be able to get something like a 3.5" drive in a caddy in there.

**Comment by zoom314** [comment]
Ok I just bought this 2000 eatx motherboard, I chose an older atx case, an AOpen brand Desktop, it has a 235w psu and 7 drive bays, of the 7 3 are 5.25" and 4 are 3.5", 2 of the 3.5" are on the outside, of the inside ones, 1 is back next to the psu and the last is up with the 2 external 3.5" drive bays. the case also came with a 3.5" 1.44mb FDD, a Seagate Medalist 2.1GB 3.5" Eide hdd and a 52x Eide cd rom drive that has some corrosion near the front of the drive and no corrosion in the case, of course I have to test everything(these came in October, the water heater died after I replaced my car which died in early Nov 2023), next will be the custom chips, then the electrolytic smd capacitors, I still need a silicone mat to solder on, but I have time, right now catching up on my insurance membership and getting a point of use water heater bought & installed so I can have hot water again is my current priority, but I have time. Yes I do need to clean the front and then paint it white, brightening the plastic seems to last only 6 months. On the metal cover I used one of the Mr Clean white sponges on the dirt, and some paint came off too, just add water to the white "sponge" and you just follow the directions.
Jason that isn't huge. I later learned this AOpen case would need sheet metal removed from the bottom drive bay to fit this motherboard, I have a Corsair Air 540 case that will fit this motherboard, it's that or a Cooler Master HAF XB Evo Case which is ATX but is wide and deep enough for an EATX motherboard.
![CashForJunkCars-164 030](https://github.com/jasonsbeer/Amiga-2000-EATX/assets/108151204/46f3b932-8412-4db4-9722-bc3e62d3a893)

**Comment by 1stFalloutBoy** [comment]
With regards to this board how many of the pcie slots on the back of the EATX case are used?
